### ISSUE TITLE
addon radio.de: v1.1.4 Add a country-language

### ISSUE THREAD

**Issue by ptodic** [fivebanger/kodi_addons#13]
Hi,

thank you for great plugin!! I would like to listen to Hungarian and Serbian radios but I cant find them anywhere, neither by country or language. I tried looking in the code but couldnt find it.  Can you please tell me how to add them.
Thanks!

**Comment by fivebanger** [comment]
The add-on can only access the services provided by radio.net. Available countries are listed here:
```
https://www.radio.net/country-selector
```

All countries listed by radio.net are available for selection in the add-on settings. If your preferred country is not listed, there is nothing I can do and there is no way to add something in the code. Sorry.

**Comment by ptodic** [comment]
In the meantime I found the solution. For the full list of available countries change in code:

        api_result = self.radio.get_countries(self.min_cnt)

 to 
        api_result = self.radio.get_countries()

**Comment by fivebanger** [comment]
You can change the self.min_cnt value of the code in the add-on settings:
```
settings->Radio->Minimum count of stations in result
```
The setting just prevents to get flooded by a lot of results which are maybe less relevant. If this fixes your problem, pls. change the setting to any value (maybe 0) which fits best for your needs.

**Comment by ptodic** [comment]
You should not put self.min_cnt on countries or languages because it will show you just a handfull (20-30?) of countries instead of all countries available. Same for languages. Probably some people are not using you addon because they couldnt find their native radios in Browse by country or Browse by language. Please consider this in next version.

Best regards and keep up the good work!

**Comment by fivebanger** [comment]
Thanks. 

The list for countries grows from 45 to 171 result entries when changing the min_cnt value to 0. The list of languages from 21 to  122. Reason is: A lot of countries and languages which gets listed then just provides a small number of stations.

So I think it's a tradeoff between visibility and convenience.

**Comment by fivebanger** [comment]
Fixed with v1.1.6
